docs/cloud-providers/add-cloud-providers-credentials.md

```diff
@@ -13,6 +13,16 @@ links:
     featured:
 ---
 
+# Generating Cloud Providers Credentials
+## GCP Service Account Credentials
+1. After creating your project, got to GCP IAM manager and select the `Service Accounts` option on the left menu
```

Comment:
> After creating your project
Devopness project? GCP project?

Shouldn't we be more direct to the user on the basic requirements?
If a user does not even have a project on her GCP account, very likely the person is not yet ready to provision resources on their account not even manually.

Maybe something like:
```
Access your GCP project or [create a new project on GCP](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
```
?
This way we give users sources for getting into deeper details and remove from us the responsibility of training users on GCP specifics.

Reply:
Suggestion accepted: https://github.com/devopness/devopness/blob/614e3070a24c883a787e519c4821679fe2bbfc67/docs/cloud-provider-credentials/get-gcp-service-account-credentials.md?plain=1#L17